csharp/ql/consistency-queries/qlpack.yml

```diff
@@ -1,6 +1,5 @@
 name: codeql-csharp-consistency-queries
-version: 0.0.0
-libraryPathDependencies:
-  - codeql/csharp-all
-  - codeql/csharp-queries
+dependencies:
+  codeql/csharp-all: "*"
+  codeql/csharp-queries: "*"
```

Comment:
Hmm, I am not sure that this line is needed.

Reply:
I'm not sure either. We can commit this suggestion if you would like to try and remove it?

```suggestion
```

Reply:
Yes, let's give it a try.